Tools/Pipeline/Gtk/Dialogs/TextEditorDialog.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Gtk;
+
+namespace MonoGame.Tools.Pipeline
+{
+    public partial class TextEditorDialog : Dialog
```

Comment:
I am trying not to change existing terminology Add File/New File, so Add Folder/New Folder